tests/acceptance/test_acceptance/test_ups.py

```diff
@@ -0,0 +1,148 @@
+import json
+
+import pytest
+
+from tests.acceptance.helpers import ENDPOINT_DECIDE, ENDPOINT_LOOKUP, ENDPOINT_SAVE
+from tests.acceptance.helpers import create_and_validate_request_and_response
+
+
+expected_flag_key = """
+    {
+      "variationKey": "variation_1",
+      "enabled": true,
+      "ruleKey": "feature_2_test",
+      "flagKey": "feature_2",
+      "userContext": {
+        "userId": "matjaz",
+        "attributes": {
+          "attr_1": "hola"
+        }
+      },
+      "reasons": []
+    }
+"""
+
+expected_after_saved_variation = """
+    {
+      "variationKey": "variation_2",
+      "enabled": true,
+      "ruleKey": "feature_2_test",
+      "flagKey": "feature_2",
+      "userContext": {
+        "userId": "user1",
+        "attributes": {
+          "attr_1": "hola"
+        }
+      },
+      "reasons": []
+    }
+"""
+
+
+@pytest.mark.parametrize(
```

Comment:
You could probably not use parameterization because you only test one case.
initially I used parameterization to combine several similar scenarios together, but here I think it would be cleaner if you just pass parameters into the funct signature